app/src/test/java/org/simple/clinic/bp/entry/confirmremovebloodpressure/ConfirmRemoveBloodPressureDialogControllerTest.kt

```diff
@@ -0,0 +1,49 @@
+package org.simple.clinic.bp.entry.confirmremovebloodpressure
+
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.verify
+import com.nhaarman.mockito_kotlin.whenever
+import io.reactivex.Completable
+import io.reactivex.Single
+import io.reactivex.subjects.PublishSubject
+import org.junit.Before
+import org.junit.Test
+import org.simple.clinic.bp.BloodPressureRepository
+import org.simple.clinic.bp.entry.ConfirmRemoveBloodPressureDialog
+import org.simple.clinic.patient.PatientMocker
+import org.simple.clinic.widgets.UiEvent
+
+class ConfirmRemoveBloodPressureDialogControllerTest {
+
+  lateinit var bloodPressureRepository: BloodPressureRepository
+  lateinit var controller: ConfirmRemoveBloodPressureDialogController
+  lateinit var dialog: ConfirmRemoveBloodPressureDialog
+
+  val uiEvents = PublishSubject.create<UiEvent>()
+
+  @Before
+  fun setUp() {
+    bloodPressureRepository = mock()
+    dialog = mock()
```

Comment:
New pattern for creating mocks in `setUp()`?

Reply:
As in? I thought we'd agreed to create mocks in `setUp` because they don't automatically get reset after every test.